pkg/images/consts.go

```diff
@@ -19,5 +19,5 @@ package images
 const (
 	SchedulerPluginSchedulerDefaultImage  = "k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.19.9"
 	SchedulerPluginControllerDefaultImage = "k8s.gcr.io/scheduler-plugins/controller:v0.19.9"
-	ResourceTopologyExporterDefaultImage  = "quay.io/openshift-kni/resource-topology-exporter:v0.0.20"
+	ResourceTopologyExporterDefaultImage  = "quay.io/openshift-kni/resource-topology-exporter:v0.2.0"
```

Comment:
let's bump to the 0.2.1 which I just pushed, to make sure we are fully aligned with the latest and greatest.